src/main/webapp/assets/js/util/brooklyn-utils.js

```diff
@@ -113,6 +113,7 @@ define([
         if ($input.attr("type") === "checkbox") {
             return $input.is(":checked");
         } else {
+            // TODO: get codemirror.getValue() in case this is a textarea ...
```

Comment:
comment from @azbarcea ... no further logic requored as all logic is on composer tab will remove comment